packages/dso-toolkit/src/components/info/info.mixins.scss

```diff
@@ -14,6 +14,10 @@
     background-color: transparent;
     border-style: none;
     padding: 0;
+  }
+
+  > button,
+  > dso-icon-button {
```

Comment:
Ook hier geldt weer dat de mixins nu teveel weten: De mixins zijn implementatie agnostisch.

We hebben twee methodes:
1. De selector meegeven vanuit de implementatie (`@include info.root("dso-icon-button");`)
2. De styling in een mixin onderbrengen en de selector bij de implementatie houden (`dso-icon-button { @include info.button-styling(); }`.

Ik twijfel: Of niets doen, of voor optie 1 gaan. Optie 2 is het meeste werk.